plugins/catalog-backend/src/schema/openapi.generated.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2023 The Backstage Authors
+ * Copyright 2024 The Backstage Authors
```

Comment:
```suggestion
 * Copyright 2023 The Backstage Authors
```

As long as not all the content is replaced this should stay as the original year, because the rest if just derivative work of that initial copyright.

Reply:
There's a prettier issue here where since we regenerate the code everytime, the block comment at the top of the generated code gets evaluated by prettier. Not sure what the best approach is here, same thing on the template side.

Reply:
Ah alright, let's not block this PR on that. Could be nice to keep the year in place but that's low prio. Not actually sure what is the correct behavior here is anyway